src/App.jsx

```diff
@@ -14,70 +15,82 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
+  const toggleGood = (selected, thisGood) => {
+    if (selected === thisGood) {
```

Comment:
`thisGood` is not correct name, because this is a service word, so better to rename it to `currentGood` or `good`